static/searchConfig.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "label": "SHAREVDE ALBERTA",

```

Comment:
https://media1.giphy.com/media/ii8rlXOlRFp5LKAVUR/giphy.gif

Reply:
Or that he copied the existing config ...

Reply:
But it's hard to read. Can we improve it by using normal case?

Reply:
@michelleif might want these to say something entirely different, e.g., different case, different names, no SHAREVDE. Would prefer to handle in a separate PR in which she can edit as she feels appropriate.